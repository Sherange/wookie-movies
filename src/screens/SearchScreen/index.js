import React from 'react';
import {View, Text, StyleSheet} from 'react-native';
import {backgroundColor, primaryTextColor} from '../../constants/theme';

const SearchScreen = () => {
  return (
    <View style={styles.viewStyle}>
      <Text style={styles.textStyle}>SearchScreen</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  viewStyle: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: backgroundColor,
  },
  textStyle : {
    fontSize : 24,
    color : primaryTextColor
  }
});

export default SearchScreen;
