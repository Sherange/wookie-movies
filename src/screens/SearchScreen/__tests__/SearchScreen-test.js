import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import SearchScreen from '../index';

it('renders correctly', () => {
  renderer.create(<SearchScreen />);
});

test('SearchScreen snapshot', () => {
  const snapshot = renderer.create(<SearchScreen />).toJSON();
  expect(snapshot).toMatchSnapshot();
});
