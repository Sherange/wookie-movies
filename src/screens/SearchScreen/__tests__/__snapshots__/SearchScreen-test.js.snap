// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchScreen snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0F1B2B",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 24,
      }
    }
  >
    SearchScreen
  </Text>
</View>
`;
